### DISCUSSION TITLE
Welcome to flagpole Discussions!

### DISCUSSION THREAD

**Discussion by jasonbyrne** [flagpolejs/flagpole#125] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnsickels** [comment]
Hello I am developer

**Comment by johnsickels** [comment]
Can we get @russellbrewer added as a contributor so he can review my PRs? @jasonbyrne

**Comment by russellbrewer** [comment]
Hi I'm Rusty and I make sure John doesn't commit any typos
